src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -227,55 +228,86 @@ public void InitializeService(ServiceHost serviceHost)
         /// Runs the async task that performs the scripting operation.
         /// </summary>
         private void RunSelectTask(ConnectionInfo connInfo, ScriptingParams parameters, RequestContext<ScriptingResult> requestContext)
-        {            
-            ConnectionServiceInstance.ConnectionQueue.QueueBindingOperation(
-                key: ConnectionServiceInstance.ConnectionQueue.AddConnectionContext(connInfo, "Scripting"),
-                bindingTimeout: ScriptingOperationTimeout,
-                bindOperation: (bindingContext, cancelToken) =>
+        {
+            Task.Run(() =>
```

Comment:
>Task.Run [](start = 12, length = 8)

queuing an operation already creates a task with ContinueWithOnFaulted and it looks like this function is already in a try\catch.

Where do you anticipate faults occurring that require this task\continuewithonfaulted wrapper?

Reply:
...on second thought I guess you're just trying to make the call actually be async, since we currently do a `queueItem.ItemProcessed.WaitOne();` which blocks, if that's the case then disregard, if the task is for error handling the comment still applies :).